scripts/qa_crowbarsetup.sh

```diff
@@ -98,6 +98,8 @@ export want_postgresql=${want_postgresql:-1}
 if iscloudver 6plus ; then
     export CROWBAR_EXPERIMENTAL=true
     export CROWBAR_VERIFY_SSL=false
+    # set timeout to 10min to avoid bsc#988030
+    export CROWBAR_TIMEOUT=600
```

Comment:
@dirkmueller thanks for rereading my comment.

I said that we need to change it for SAP to be longer than the old timeout. We have builds of SAP PRs where a timeout hits after about 5 minutes. That is too short, so we need to be able to raise this.

What the the old value is (or values are) is currently discussed here.